### ISSUE TITLE
Rename default table used in query caching

### ISSUE THREAD

**Issue by DarkInsight85** [typeorm/typeorm#3993]
**Issue type:**

[X] question
[ ] bug report
[X] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[X] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
[X] `0.2.7` (or put your version here)

Hi, for the default table "query-result-cache" used in caching, is there a way to rename this table?

**Comment by Kononnable** [comment]
No, it's not possible right now but it should be easy to make a PR with such enhancement if you need it. 
`src/cache/DbQueryResultCache.ts` is the file where this name is defined. If you want to change it to be customizable there should be another property in connection options through which new name could be passed.

**Comment by YehezkielSaputra** [comment]
@Kononnable Just for your information, I have created some changing based on this issue but I have no access to push my commit and create Pull Request.
Would you please help me solve this issue?
Any of your help would be appreciated.
